File: 面试题17.14/面试题17.14.cpp
// Author: washing
// DateTime: 2021/9/3 18:50
// File: 面试题17.14.cpp
// Desc: 


class Solution {
public:
    vector<int> smallestK(vector<int>& arr, int k) {
        vector<int> vec(k, 0);
        sort(arr.begin(), arr.end());
        for (int i=0; i < k; i++) vec[i] = arr[i];
        return vec;
    }
};

File: 面试题17.14/面试题17.14.py
# -*- coding:utf-8 -*-
# Author: washing
# DateTime: 2021/9/3 14:07
# File: 面试题 17.14.py
# Desc: 

class Solution:
    def smallestK(self, arr: List[int], k: int) -> List[int]:
        arr.sort()
        return arr[:k]